docs/reference/configuration/data_types.rst

```diff
@@ -72,167 +61,352 @@ This section only applies if a fixed seed is used:
   * Parameter order is irrelevant
   * Parameter names are case sensitive, PARAM:MY_PARAM != PARAM:myParam
 
-NORMAL
-,,,,,,
+NORMAL: Gaussian Distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To set a normal (Gaussian) prior, use the keyword NORMAL. It takes two
-arguments, a mean value and a standard deviation. Thus, the following example
-will assign a normal prior with mean 0 and standard deviation 1 to the variable
-VAR1:
+The ``NORMAL`` keyword allows assigning a Gaussian (or normal) prior to a variable.
+It requires two arguments: a mean value and a standard deviation.
 
+Syntax
+~~~~~~
 ::
 
-   VAR1   NORMAL    0   1
+  VAR NORMAL <mean_value> <standard_deviation>
+
+Parameters
+~~~~~~~~~~
+- **<mean_value>**: Specifies the mean of the normal distribution.
+- **<standard_deviation>**: Indicates the standard deviation of the normal distribution.
+
+Example
+~~~~~~~
+For a Gaussian distribution with mean 0 and standard deviation 1 assigned to the variable VAR:
+::
+
+   VAR NORMAL 0 1
+
+.. image:: fig/normal.png
+
+Notes
+~~~~~
+The ``NORMAL`` keyword is integral for scenarios demanding priors that reflect typical real-world data patterns, as the Gaussian distribution is prevalent in many natural phenomena.
 
-LOGNORMAL
-,,,,,,,,,
+LOGNORMAL: Log Normal Distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A stochastic variable is log normally distributed if the logarithm of the
-variable is normally distributed. In other words, if X is normally distributed,
-then Y = exp(X) is log normally distributed.
+The ``LOGNORMAL`` keyword is used to assign a log normal prior to a variable. A variable is considered log normally distributed if the logarithm of that variable follows a normal distribution. 
+In mathematical terms, if :math:`X` is normally distributed, then :math:`Y = e^X` is log normally distributed.
 
-A log normal prior is suited to model positive quantities with a heavy tail
-(tendency to take large values). To set a log normal prior, use the keyword
-LOGNORMAL. It takes two arguments, the mean and standard deviation of the
-*logarithm* of the variable:
+Log normal priors are especially suitable for modeling positive quantities that exhibit a heavy tail, indicating a tendency for the quantity to occasionally take large values.
 
+Syntax
+~~~~~~
 ::
 
-   VAR2   LOGNORMAL  0  1
+  VAR LOGNORMAL <log_mean> <log_standard_deviation>
 
-TRUNCATED_NORMAL
-,,,,,,,,,,,,,,,,,
+Parameters
+~~~~~~~~~~
+- **<log_mean>**: Represents the mean of the logarithm of the variable.
+- **<log_standard_deviation>**: Denotes the standard deviation of the logarithm of the variable.
+
+Example
+~~~~~~~
+For a variable with its logarithm having a mean of 0 and a standard deviation of 1:
+::
+
+   VAR LOGNORMAL 0 1
 
-This *TRUNCATED_NORMAL* distribution works as follows:
+.. image:: fig/lognormal.png
```

Comment:
How about "Histogram from values sampled from a lognormal variable specified with log-mean of 0 and log standard deviation 1."